src/Remapping/distributed_remapping.jl

```diff
@@ -922,13 +945,17 @@ end
 """
        interpolate(field::ClimaCore.Fields;
                    hresolution = 180,
-                   zresolution = 50,
+                   zresolution = nothing,
                    target_hcoords = default_target_hcoords(space; hresolution),
-                   target_zcoords = default_target_vcoords(space; zresolution)
+                   target_zcoords = default_target_zcoords(space; zresolution)
                    )
 
 Interpolate the given fields on the Cartesian product of `target_hcoords` with
-`target_zcoords` (if not empty).
+`target_zcoords`. For `Space`s without horizontal/vertical component, the relevant
+`target_*coords` are ignored.
+
+When `zresolution` is set to `nothing`, use the Space levels, otherwise perform linear
+interpolation.
```

Comment:
In the `nothing` case, there is no vertical interpolation, correct? It might be more clear to explicitly say that.